Traceback (most recent call last):
  File "/home/aih/sebastian.birk/workspace/projects/nichecompass-reproducibility/scripts/reference_query/../map_query_on_nichecompass_reference_model.py", line 220, in <module>
    reference_model = NicheCompass.load(
  File "/home/aih/sebastian.birk/workspace/projects/nichecompass/nichecompass/models/basemodelmixin.py", line 206, in load
    load_saved_files(dir_path,
  File "/home/aih/sebastian.birk/workspace/projects/nichecompass/nichecompass/models/utils.py", line 69, in load_saved_files
    raise ValueError("Dir path contains no saved anndata and no adata was "
ValueError: Dir path contains no saved anndata and no adata was passed.
